persistence/docs/CHANGELOG.md

```diff
@@ -7,13 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.0.23] - 2023-01-18
+
+- Consolidate the `Block` proto structures under `shared/core/types`
```

Comment:
Same as previous comment, I think this should change:

`Remove block proto structure to consolidate under shared/core/types`.